test/zdtm.py

```diff
@@ -1515,6 +1515,7 @@ def cr(cr_api, test, opts):
     for i in iters[0]:
         pres = iter_parm(opts['pre'], 0)
         for p in pres[0]:
+            sbs('iter')
```

Comment:
this is pre-pre-dump :)

Reply:
Yes =) Hope its ok to just call it `iter`.

Reply:
Could you also print the number of the iteration?

Reply:
> Yes =) Hope its ok to just call it iter.

I think we need to use a different name for this stop point. Here is another loop that enumerates iterations.

Reply:
>> Yes =) Hope its ok to just call it iter.
>
> I think we need to use a different name for this stop point. Here is another loop that enumerates iterations.

Looks like the only way to fix naming here is to replace "pre-" with "before" in places where "pre-" is not related to criu-pre-dump. -> Done.